pyproject.toml

```diff
@@ -103,6 +103,10 @@ base = [
     "anomalib[core]==1.1.0",
 ]
 
+transformers = [
```

Comment:
```suggestion
huggingface = [
```
It's for enabling huggingface, so how about to change like this?

Reply:
This is ambiguous, but the library is actually a `transformers` integration. It's an integration of the transformers api, which supports the hugging face model, so I think transformers makes more sense for me. What do you think?

Reply:
I see. my thought is that the goal of integrating a `transformers` is for using huggingface model and name `transformers` can mean various thing. So, I think when user wants to use huggingface model, using `pip install otx[huggingface]` is more intuitive. This is just my thought. If you think different please resolve this :)

Reply:
Yeah we'll have to think about this a bit more, thanks for the comment!